package/darwin/generate_package_contents

```diff
@@ -123,6 +123,25 @@ gem install --platform ruby "${vagrant_gem}" || exit
 rm -rf "${embed_gemsdir}/gems/grpc-"*/src/ruby/ext/grpc/libs
 rm -rf "${embed_gemsdir}/gems/grpc-"*/src/ruby/ext/grpc/objs
 
+# Remove cached gems
+rm -f "${embed_gemsdir}/cache/"*.gem
+
+# Remove binary from fsevent as it uses an SDK that is unsupported
+rm -f "${embed_gemsdir}/gems/rb-fsevent"*/bin/fsevent_watch
```

Comment:
Does this end up hurting any functionality?

Reply:
No, it's unused